pkg/kubelet/rktshim/imagestore.go

```diff
@@ -30,32 +34,87 @@ var (
 var _ container.ImageService = (*ImageStore)(nil)
 
 // ImageStore supports CRUD operations for images.
-type ImageStore struct{}
+type ImageStore struct {
+	rkt            CLI
+	requestTimeout time.Duration
+}
 
 // TODO(tmrts): fill the image store configuration fields.
-type ImageStoreConfig struct{}
+type ImageStoreConfig struct {
+	CLI            CLI
+	RequestTimeout time.Duration
+}
 
 // NewImageStore creates an image storage that allows CRUD operations for images.
-func NewImageStore(ImageStoreConfig) (container.ImageService, error) {
-	return &ImageStore{}, nil
+func NewImageStore(cfg ImageStoreConfig) (container.ImageService, error) {
+	return &ImageStore{rkt: cfg.CLI, requestTimeout: cfg.RequestTimeout}, nil
 }
 
 // List lists the images residing in the image store.
-func (*ImageStore) List() ([]container.Image, error) {
-	panic("not implemented")
+func (s *ImageStore) List() ([]container.Image, error) {
```

Comment:
@tmrts OK for now, but always not a fan of using CLI in codes. I am starting working on the image service implementation in rkt api service.